pkg/ibmvpc/vpc.go

```diff
@@ -233,10 +235,22 @@ func (lb *LoadBalancer) AddressRange() *ipblock.IPBlock {
 	return nodesAddressRange(lb.nodes)
 }
 
-// we do not need this func, for now it is here since the linter warn that lb.listeners are not in use
-// todo - remove:
-func (lb *LoadBalancer) NListeners() int {
-	return len(lb.listeners)
+// AllowConnectivity() - check if lb allow connection from src to dst
+// currently only a boolean function, will be elaborated when parsing policies rules
+func (lb *LoadBalancer) AllowConnectivity(src, dst vpcmodel.Node) bool {
+	// currently we do not allow connections from privateIP to a destination that is not a pool member
```

Comment:
```suggestion
	// currently we only allow connections from privateIP to a destination that is a pool member
```

Reply:
its not true
we also allow connection from a non-privateIP to the privateIPs

Reply:
please add documentation explaining what is allowed

Reply:
After discussing - there are certain connections that given the load balancer are not possible. These are denied, currently initial implementation.  Please clarify in doc and naming,